src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -1004,4 +1004,281 @@ class Project: ADOSVTBase
         }
         return $controlResult
     }
+
+    hidden [ControlResult] CheckInactiveProject([ControlResult] $controlResult) {
+
+        # these variables are used to make table which will show inactivity status for resource types.
+        $Reporow = @()
+        $Buildrow = @()
+        $Releaserow = @()
+        $AgentPoolrow = @()
+        $ServiceConnectionrow = @()
+
+        ## Checking Inactive Repos
+        $IsRepoActive = $false
+        try {            
+            $repoDefnsObj = $this.FetchRepositoriesList()
+            $inactiveRepos = @()
+            $threshold = $this.ControlSettings.Repo.RepoHistoryPeriodInDays
+            if (-not ($repoDefnsObj.Length -eq 1 -and [Helpers]::CheckMember($repoDefnsObj,"count") -and $repoDefnsObj[0].count -eq 0)) 
+            {
+                foreach ($repo in $repoDefnsObj) {
+                    $currentDate = Get-Date
```

Comment:
Why are we considering repos inactive even if only 1 repo is inactive while others are active? 
Please check with Manish if this the correct approach

Reply:
As discussed with Manish, made changes if one repo is active then repo activity status will be true